prime-router/settings/organizations-prod.yml

```diff
@@ -9,6 +9,7 @@
     - name: default
```

Comment:
Sorry we didn't catch this sooner - we are actually thinking of deleting this and probably organizations-test.yml too.   See conversation here:  https://usds.slack.com/archives/C01HFHPNJEL/p1632227581289000

Reply:
Should I remove the changes from this, or leave them because it doesn't matter?

Reply:
seems fine to leave them